computation_graph/run.py

```diff
@@ -73,18 +60,13 @@ def _transpose_graph(
 
 def _get_computation_input(
     node: base_types.ComputationNode,
-    incoming_edges: base_types.GraphType,
-    results: Tuple[Tuple[base_types.Result, ...], ...],
+    edge_to_result: Dict[base_types.ComputationEdge, Tuple[base_types.Result, ...]],
```

Comment:
wdym by edge to result? every node has a result, not every edge. am i wrond?

Reply:
each node has  a result, and it flows through an edge